topics/lua-api.md

```diff
@@ -515,9 +518,20 @@ The reply's format is `MM.mm.PP`, where:
 * **mm:** is the minor version.
 * **PP:** is the patch level.
 
-### <a name="redis.redis_version_num"></a> `redis.REDIS_VERSION_NUM`
+### <a name="server.redis_version"></a> `server.REDIS_VERSION`
+
+* Available in scripts: yes
```

Comment:
If called as `redis.REDIS_VERSION`, it came in 7.0.0.

```suggestion
### <a name="server.redis_version"></a> `server.REDIS_VERSION`

* Since version: 7.0.0
* Available in scripts: yes
```